galley/queries.py

```
@@ -129,3 +129,46 @@ def get_formatted_recipe_ingredients(recipe_id) -> Optional[Dict]:
         return {'ingredients': main_ingredients, 'standalone_ingredients': standalone_ingredients}
     else:
         return None
+
+
+def get_formatted_menu_data(names: list) -> Optional[List[Dict]]:
+    menus = get_week_menu_data(names)
+    formatted_menus = []
+
+    if not menus:
+        return None
+
+    for menu in menus:
+        formatted_menu = ({
+            'name': menu.get('name'),
```

do we want getting the keys here to fail silently? for example if 
`a = {}` `a.get('name')` would just return `None` hiding the issue that the data is missing the key. if we are expecting the data to contain certain keys we should have it throw KeyError so that we can handle that exception. here's an example.

```
try:
    name = menu['name']
except KeyError:
    logger.error('menu data for date x is missing key "name"')
    name = ""
```

this way the method would still return blank values if key does not exist but if we were investigating why menu data is missing a certain piece of data we can actually check the logs.

if you think a key is crucial such then the method should just raise an exception and not return a blank value.

---

This makes sense in case it fails on our end. For context, it's required that all the fields within `formatted_menu` (minus `date`) are non-nullable on the Galley side.

---

I'm not necessarily sure if we should have them return as empty strings, but I could be wrong! @mollykmcglone What do you think?

---

I think this is a case where we want to support the happy path for now and come back later to add handling for missing data. We will need to do the same at the recipe level as well.

---

sounds good to me, we are checking validity in a method at a higher level so I'm fine with that